### DISCUSSION TITLE
[Question] Isaac Lab Installation and Vscode Setup

### DISCUSSION THREAD

**Discussion by evanzijianhe** [isaac-sim/IsaacLab#620] [Q&A]
I installed the Isaac Sim by using pip install method, and also installed the Isaac Lab following the instruction. I was able to run the testing code to verify the installation. However, I have some questions regarding Vscode setup:

1. Do we still need to run the `setup_python_env` in the IsaacLab and IsaacLabExtensionTemplate if I was using pip install? I tried, it was impossible to properly execute it, because `_isaac_sim` symbolic link was missed. Does it mean we still need to install the IsaacSim using omniuniverse-launch and setup path for `_isaac_sim`? 
2. Can we install IsaacSim use both pip and pre-built binaries methods? 
3. When I import functions from omni.isaac.lab class, those function are unknown( but program can be executed). Is this normal? 
![Screenshot from 2024-07-02 15-41-10](https://github.com/isaac-sim/IsaacLab/assets/41100228/de944e0a-f8f6-4821-bdde-0bf79c5bd88a)

Can someone helped to answer those questions? Thank you very much!

**Comment by Mayankm96** [comment]
Hi,

Thank you for reporting this. I made some fixes in this MR: https://github.com/isaac-sim/IsaacLab/pull/628

Could you please confirm this works for you as well?

**Reply by evanzijianhe** [reply]
Hi Mayank. Thanks for your helps. I have tested this fix. There was no `.vscode` fold in the isaacsim fold in anaconda, so I could not find `anaconda3/envs/isaaclab/lib/python3.10/site-packages/isaacsim/.vscode/settings.json`.

However, I found that the `settings.json` file exits in the `isaacsim/extscache/omni.kit.window.extensions/ext_template/vscode/settings.json`, so I manually created a .vscode fold in the root of isaacsim folder and copy the settings.json to there. Then, everything is fine.

**Reply by evanzijianhe** [reply]
After doing above setup, I found 
there is a error in the vscode which was caused by `tuple`: 
![Screenshot from 2024-07-03 22-49-32](https://github.com/isaac-sim/IsaacLab/assets/41100228/5f988d52-fcf6-4e2e-985b-9a916b2dfd76)
replace `[ ]` by `( )` can fix
![Screenshot from 2024-07-03 22-50-06](https://github.com/isaac-sim/IsaacLab/assets/41100228/1ada71a2-c0ea-4420-bf03-6848747795f8)
